### ISSUE TITLE
Msal UserAgentApplication.loginPopup , Error opening popup window

### ISSUE THREAD

**Issue by traoreba** [AzureAD/microsoft-authentication-library-for-js#1825]
>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library
- [x] `msal@1.3.1` 

## Framework
React JS

## Description
Hi, I'm using msal for user authentication in my Teams Tab. It work fine in Teams Web on google chrome but when I use it on Teams desktop app I have the error `Error opening popup window. This can happen if you are using IE or if popups are blocked in the browser.`
Thanks for your help.

## Error Message
Error opening popup window. This can happen if you are using IE or if popups are blocked in the browser.

## MSAL Configuration

```js
// Provide configuration values here.
// For Azure B2C issues, please include your policies.
```

## Reproduction steps
```js
this.userAgentApplication = new UserAgentApplication({
      auth: {
          clientId: appId,
          redirectUri: window.location.origin
      },
      cache: {
          cacheLocation: "sessionStorage",
          storeAuthStateInCookie: true
      }
    });
try {
        authRes = await this.userAgentApplication.loginPopup(
          {
            scopes: this.inTeams ? config.scopesInTeams : config.scopes,
            prompt: "select_account"
        });
}catch (err) {
console.error(err);
}
```

## Browsers/Environment
- [x] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Safari
- [ ] IE
- [ ] Other (Please add browser name here)

**Comment by jmckennon** [comment]
`loginPopup` will work on the web version of teams, but for the desktop version you'll need to call a special Teams API which creates a Teams popup, and then call `loginRedirect()` from there. There was a discussion and fix around this subject in #1095 and a [user even create an example](https://github.com/nmetulev/teams-msal/blob/master/index.js#L60).

**Comment by traoreba** [comment]
Thanks @jmckennon, it worked fine
